src/Parser/Core/Modules/DamageMitigation/AOE_ABILITIES.js

```diff
@@ -0,0 +1,50 @@
+// White list of all AOE abilities (INCOMPLETE!)
+export default 
+[
+  // Garothi World Breaker
+  {
+    id: 244761,
```

Comment:
Please include the spell name in a comment.

Reply:
Will do in the future, was just being lazy with the test implementation. On that note, I later realized that there is a good chance that ticks are never reduced by avoidance, in which case the whitelist could simply be an array of ids. I never got around to test that theory though.